docs/getting-started.md

```diff
@@ -9,65 +9,234 @@ sidebar_label: Getting started
+
+While this is nothing new in itself -- P2P architectures have existed since the 1990’s (where they rose to fame with file sharing programs like BitTorrent and Napster) -- what's new is the addition of cryptography and economic incentives to these architectures.
 
-What you need to get started building with Aragon:
+This fusion of these seemingly disparate disciplines was the big innovation behind Bitcoin, and has since led to the emergence of a new field of research devoted to their intersection (what we now call cryptoeconomics). 
 
-- [Environment setup](#environment-setup)
-- [Quick start](#quick-start)
-- [Next steps](#next-steps)
+While we won't get into the details here,the key takeaway is that cryptoeconomics is the big unlock that has allowed us to start moving from centralized data structures (web2) to more decentralized or fully distributed data architectures (web3).
+
+![](/docs/assets/centralized-vs-decentralized-stack-2.png)
+
+> Note that there’s a spectrum from fully centralized (left) to fully decentralized (right).
+
+And while blockchains -- like Bitcoin and Ethereum -- are key to this web3 vision, it's important to note that there are other essential parts of the web3 stack that are not covered by them. For example, since blockchains are relatively expensive to store data on, it turns out that they don't make great file systems. That's why there's also a need for decentralized file systems like the [InterPlanetary File System](https://ipfs.io/) (which Aragon also makes use of).
+
+
+### Further reading
+...
+
+
+# Up and running
+
+Now that we've got you all excited, let's go through what you need to get started building with Aragon 😊
 
 
 ## Environment setup
 
-### Node version
+### Node.js
+
+First off, we need to sure we have a recent version of Node.js installed, at least `v8.0.0`.
+
+To see which version of Node you have installed, from the command line run:
+
+```sh
+node -v
+```
+
+If you need to download a more recent version, [see here](https://nodejs.org/en/download/).
+
+
+### Web3 provider
+
+Next, we'll need what we call a web3 provider to actually sign and send transactions to the Ethereum blockchain.
+
+We recommend either [Frame](https://frame.sh) or [Metamask](https://metamask.io/).
+
+If you're new to the decentralized web you might be wondering why we have to use a separate provider to interact with the blockchain. Why don't decentralized apps (like Aragon's) just do it themselves?
+
+In short, while it's possible for dapps to interact directly with the blockchain, using a web3 provider allows users to interact with dapps without trusting every one of them with their private keys (the keys to theirs funds). Without a web3 provider, users have to have total trust in every dapp they use. With a web3 provider, they just need to trust that provider.
+
+#### Metamask 
```

Comment:
I think we should write a little paragraph for each provider and then:
1. For Metamask link an external tutorial.
2. For Frame link this [soon to be made guide](https://github.com/aragon/hack/issues/106)

Reply:
Perfect. Will find a decent Metamask tutorial. Let me know if I can help with the guide.

Reply:
So I've had a good look, and I'm not completely happy with any of the existing Metamask tutorials. I suggest we make our own guide for Metamask too. What do you think? Happy to take care of it.

Reply:
Not sure, I think Aragon goal is to promote Frame as the go to provider to interact with DAOs. At the same time we can't deny that Metamask is the first option for most people right now. 

So, we could start defining the scope for both guides and further explain Frame cause there is almost no documentation avaible (in comparation with Metamask)

Reply:
Sounds good.

Even if we have a Metamask guide, I think we can nudge the reader into using Frame by explaining what the advantages are and displaying it first.

Reply:
Yes! I like that plan